build/board/NXPEVK_iMX8M_4GB/NXPEVK_iMX8M_4GB_ProductionOEMInput.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<OEMInput xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://schemas.microsoft.com/embedded/2004/10/ImageUpdate">
+  <Description>Windows 10 IoT Core NXPEVK_iMX8M_4GB Test FFU generation.</Description>
+  <SOC>NXP_IMX8M</SOC>
+  <SV>NXP</SV>
+  <Device>NXPEVK_iMX8M_4GB</Device>
+  <ReleaseType>Test</ReleaseType>
+  <BuildType>fre</BuildType>
+  <SupportedLanguages>
+    <UserInterface>
+      <Language>en-us</Language>
+    </UserInterface>
+    <Keyboard>
+      <Language>en-us</Language>
+    </Keyboard>
+    <Speech>
+      <Language>en-us</Language>
+    </Speech>
+  </SupportedLanguages>
+  <BootUILanguage>en-us</BootUILanguage>
+  <BootLocale>en-us</BootLocale>
+  <Resolutions>
+    <Resolution>1024x768</Resolution>
+  </Resolutions>
+  <AdditionalFMs>
+    <AdditionalFM>%AKROOT%\FMFiles\arm64\IoTUAPNonProductionPartnerShareFM.xml</AdditionalFM>
+    <AdditionalFM>%BLD_DIR%\MergedFMs\NXPEVK_iMX8M_4GB_DeviceFM.xml</AdditionalFM>
+  </AdditionalFMs>
+<Features>
+<Microsoft>
+  <Feature>IOT_KDSERIAL_SETTINGS</Feature>
+  <Feature>IOT_EFIESP</Feature>
+  <Feature>IOT_EFIESP_BCD</Feature>
+  <Feature>IOT_APP_TOOLKIT</Feature>
+  <Feature>IOT_SHELL_HOTKEY_SUPPORT</Feature>
+  <Feature>IOT_SHELL_ONSCREEN_KEYBOARD</Feature>
+  <Feature>IOT_SIREP</Feature>
+  <Feature>IOT_SSH</Feature>
+  <Feature>IOT_UAP_DEFAULTAPP</Feature>
+  <Feature>IOT_WEBB_EXTN</Feature>
+  <Feature>IOT_DIRECTX_TOOLS</Feature>
+  <Feature>IOT_CRT140</Feature>
+  <Feature>IOT_POWERSHELL</Feature>
+  <Feature>IOT_POWER_SETTINGS</Feature>
+  <Feature>IOT_ENABLE_ADMIN</Feature>
+  <Feature>IOT_TOOLKIT</Feature>
+  <Feature>IOT_WDTF</Feature>
+  <Feature>IOT_ENABLE_TESTSIGNING</Feature>
+  <Feature>IOT_UMDFDBG_SETTINGS</Feature>
+  <Feature>IOT_USBFN_CLASS_EXTENSION</Feature>
+<!--  <Feature>IOT_ONBOARDING_APP</Feature> -->

```

Comment:
Do we want to keep all these features commented out or just remove them wholesale?

Reply:
The IOT_ONBOARDING_APP isn't available on the ARM64 platform yet.  The others are.  We don't include them for the other boards in Kayla, but they are included in the .xml files in the OS tree.

Reply:
Okay, sounds fine to leave them commented out then.